GameData/SPEngine/Parts/LR79_proc.cfg

```diff
@@ -1,40 +1,34 @@
-+PART[liquidEngine1-2]:NEEDS[!ROEngines]:AFTER[ZREALPLUME]
++PART[liquidEngine1-2]:NEEDS[!ROEngines]:FOR[SPEngine]
 {
 	@name = SPEngineLR79
 	@title = K-class Procedural Engine
 	@description = Kerolox atmospheric engine, like LR79 family (and later SpaceX Merlin).
+	@TechRequired = orbitalRocketry1956
 	MODULE
 	{
 		name = ModuleSPEngine
 		familyLetter = K
 	}
 	@identicalParts = SPEngineLR79,SPEngineH1 // we're not identical to the part we copied!
 	!engineType = // prevent ROE from deleting us
-	@MODULE[ModuleEngines*]
-	{
-		%powerEffectName = Kerolox-Lower
-	}
 }
 
-+PART[ROE-LR89]:NEEDS[ROEngines]:AFTER[ZREALPLUME]
++PART[ROE-LR79]:NEEDS[ROEngines]:FOR[SPEngine]
```

Comment:
I picked the '89 deliberately, because its model's mounting ring was slimmer, meaning that you could put them on smaller tanks (needed since we don't yet have model scaling in SPE).  Unless ROE has changed models, I'd like to keep that decision.

Reply:
ROEngines now use BDB models for a lot of the engines